.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+sudo: false
+python:
+  - "2.7"
+env:
+  - TOX_ENV=docs
+install:
+  - pip install -rrequirements/docs.txt
+script:
+  - cd docs && make html SPHINXOPTS='-nW'
```

Comment:
Ah, excellent, didn't know about that! Added the target to the make invocation, thanks!